src/SimulationControl/SpheralController.py

```diff
@@ -584,6 +595,8 @@ def loadRestartFile(self, restoreCycle,
         if self.restartFileConstructor is GzipFileIO:
             file = self.restartFileConstructor(fileName, Read)
                                                #readToMemory = True)
+        elif self.restartFileConstructor is SidreFileIO and self.SPIOFileCountPerTimeslice is not None:
+            file = self.restartFileConstructor(fileName, Read, self.SPIOFileCountPerTimeslice)
         else:
             file = self.restartFileConstructor(fileName, Read)
```

Comment:
I think this is left over from something else but it looks like GzipFileIO falls under the else category here now.

Reply:
Is that something we want to get rid of now or in its own PR for tracking?

Reply:
I think it's fine to do here unless @jmikeowen has any objections

Reply:
Are you asking if we want to get rid of GzipFileIO?  My preference is to keep it -- it's a functional FileIO implemenation that only depends on Python (used to be our default in fact).  You never know when we might be on a platform where Silo or Sidre aren't available (or those could go away).  I'd need to know why we want to remove it before I'd go for it.

Reply:
No, not remove GzipFileIO. We are just calling the same constructor signature in `if GzipFileIO` as we are in `else` so we could just have that condition fall under else.
```
if self.restartFileConstructor is SidreFileIO and self.SPIOFileCountPerTimeslice is not None:
  file = self.restartFileConstructor(fileName, Read, self.SPIOFileCountPerTimeslice)
else:
  file = self.restartFileConstructor(fileName, Read)
```